application/forms/Command/Object/ScheduleHostDowntimeForm.php

```diff
@@ -35,6 +36,14 @@ public function __construct()
 
         $flexibleDuration = $config->get('settings', 'hostdowntime_flexible_duration', 'PT2H');
         $this->flexibleDuration = new DateInterval($flexibleDuration);
+
+        $this->on(self::ON_SUCCESS, function () {
+            Notification::success(tp(
+                'Downtime scheduled successfully',
+                'Downtimes scheduled successfully',
```

Comment:
Put `Scheduled` in front. I know I've told you otherwise in chat, but I think now this sounds better.